CTFd/utils.py

```diff
@@ -791,7 +791,6 @@ def import_ctf(backup, segments=None, erase=False):
             if data:
                 saved = json.loads(data)
                 for entry in saved['results']:
```

Comment:
IMHO it might be better to avoid using `dataset` entirely for models that depend upon other models.